### PR TITLE
Add account settings page

### PR DESCRIPTION
**Pull request by doskachok**

### CODE REVIEW

**Comment by AwesomeFlax** [comment]
Great job!
Just few things here before we merge it:
- I don't see any "feedback" mechanism on saving, at least notification should be sent
- We should work on representation side of disabled inputs, there's no difference between disabled and enabled input filed, I have an idea, but that's for side PR
- please, make sure you disable buttons and input filed while saving operation is in progress 
- to be discussed: do we want do display mini-avatar here if it was set?
image

**Comment by Sujohn77** [comment]
Better to use useEffect or useQuery for safty image

**Comment by doskachok** [comment]
> Better to use useEffect or useQuery for safty image

Not sure that I understand what do you mean. Could you provide more details?

**Comment by doskachok** [comment]
@AwesomeFlax, _I don't see any "feedback" mechanism on saving, at least notification should be sent_  - now the button got disabled after sending request to the server. I guess it should be enough. e02dcb94431b840adff19c19e19c373812c4cc4f
_We should work on representation side of disabled inputs, there's no difference between disabled and enabled input filed, I have an idea, but that's for side PR_ - please see: fb8acf9756d543896b16fb7fc4af3502106b9b02
_please, make sure you disable buttons and input filed while saving operation is in progress_ - loader disables the whole screen while a request is in progress, you can test it by setting Slow 3G in the devtools

File: src/components/Input/index.styled.ts [hunk 1]
```diff
@@ -71,6 +72,10 @@ export const InputStyled = styled.input<IInputProps>`
     font-weight: 300;
     color: ${props => props.theme.colors.text.primary};
   }
+   
+  :disabled {
+    color: ${props => props.theme.colors.input.border.default};
```
Comment (apximax) [inline src/components/Input/index.styled.ts]: text color = border color
looks a bit illogically

File: src/components/Input/index.styled.ts [hunk 2]
```diff
@@ -85,5 +90,9 @@ export const InputStyled = styled.input<IInputProps>`
     ::placeholder {
       color: ${props => props.theme.colors.input.placeholder.reversed};
     };
+
+    :disabled {
+      color: ${props => props.theme.colors.input.border.reversed};
```
Comment (apximax) [inline src/components/Input/index.styled.ts]: ext color = border color
looks a bit illogically

**Review by apximax — APPROVED** [review]

File: src/components/Input/index.styled.ts (hunk 1 above)
Comment (doskachok) [reply src/components/Input/index.styled.ts]: fixed in f5a9af1f762e68cc9b15c6f3a6c0ac0104ef4767

**Pull request merged**